packages/core/admin/ee/server/register.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
 
+const { features } = require('@strapi/strapi/lib/utils/ee');
 const executeCERegister = require('../../server/register');
+const createAuditLogsService = require('./services/audit-logs');
 
 module.exports = async ({ strapi }) => {
+  if (features.isEnabled('audit-logs')) {
+    const auditLogsService = createAuditLogsService(strapi);
+    strapi.container.register('audit-logs', auditLogsService);
+    auditLogsService.bootstrap();
+  }
   // TODO: register auditLogs provider here
```

Comment:
I think we can remove this comment as that will probably be done inside of the boostrap method of the service